dev/io.openliberty.microprofile.reactive.streams.operators30.internal_fat_tck/bnd.bnd

```diff
@@ -0,0 +1,37 @@
+#*******************************************************************************
+# Copyright (c) 2018, 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+
+src: \
+	fat/src
+
+fat.project: true
+
+javac.source: 11
+javac.target: 11
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
+
+tested.features: concurrent-3.0
+
```

Comment:
Don't think this is needed? I can't see anywhere where we're using concurrent-3.0.